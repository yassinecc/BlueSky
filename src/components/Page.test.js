import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import Page from './Page';

describe('Page', () => {
  it('should render the page', () => {
    const tree = renderer
      .create(
        <Page>
          <Text>Hello</Text>
        </Page>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
