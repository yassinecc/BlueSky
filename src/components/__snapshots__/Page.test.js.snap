// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render the page 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "flex-start",
      },
      Object {
        "backgroundColor": undefined,
        "paddingHorizontal": 24,
        "paddingTop": 16,
      },
    ]
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
